lowranceusr.cc

```diff
@@ -758,24 +1652,120 @@ lowranceusr_waypt_disp(const Waypoint* wpt)
   /* USER waypoint type */
   short int WayptType = 0;
   gbfputint16(WayptType, file_out);
+
+  if (writing_version == 3) {
+    float depth = WAYPT_HAS(wpt, depth) ?
+                  METERS_TO_FEET(wpt->depth) : -99999.0;
+    gbfputint32(depth, file_out);
+  }
+
+  if (global_opts.debug_level > 1)
+    printf("\n");
 }
 
+static void
+lowranceusr4_waypt_disp(const Waypoint* wpt)
+{
+  /* UID unit number */
+  if (opt_serialnum_i > 0)
+    gbfputint32(opt_serialnum_i, file_out);   // use option serial number if specified
+  else if (wpt->fs != (format_specific_data *)NULL)
```

Comment:
nullptr instead of "(format_specific_data *) NULL" ?

Reply:
Adopted suggestion.